tests/integration/targets/vmware_cluster_info/aliases

```diff
@@ -1,4 +1,3 @@
 cloud/vcenter
 needs/target/prepare_vmware_tests
 zuul/vmware/vcenter_1esxi
-zuul/vmware/govcsim
```

Comment:
This would disable `govcsim` for all the tests of this plugin. I believe the existing ones can pass on `govcsim` just fine. I would prefer we use a `block:` to be more specific.

Reply:
Thank you for your comment!
This patch is to add the `GetResourceUsage` method to get a resource summary of the cluster.  
However, the `GetResourceUsage` method is not supported by govcsim.  
So, whenever you try to get summary information using govcsim, an error always occurs in the test, even if the test has been working properly up until now due to the following changes.

see: https://github.com/ansible-collections/vmware/blob/bfce0d5a3b2401561b3a997a0716b7454047bc35/plugins/modules/vmware_cluster_info.py#L289

Should I consider a different implementation?

Reply:
I'm assuming you don't need to use govcsim because the tests are done on a virtual environment, but what do you think?

```
zuul/vmware/vcenter_1esxi
```

Reply:
Oh I see. Ok, this makes sense indeed. Ignore my comment.